tests/test_onco_print.py

```diff
@@ -94,3 +117,25 @@ def assert_callback(prop_value, nclicks, input_value):
         component_base=COMPONENT_REACT_BASE,
         data=TEST_DATA
     )
+
+
+def test_padding(dash_threaded):
+    """Test the updating of the padding value."""
+    def assert_callback(prop_value, nclicks, input_value):
+        answer = ''
+        if nclicks is not None:
+            if float(input_value) == prop_value:
```

Comment:
@shammamah @Bachibouzouk here I hesitated with this:
```suggestion
            if PROP_TYPES['float'](input_value) == prop_value:
```
allowing the flexibility (possible future change?) of this [definition](https://github.com/plotly/dash-bio/blob/e16f5405bc052d70fbb1c2da52969c94ab55e709/tests/test_common_features.py#L44). What do you think?

Reply:
This is a great idea! We had to change the `bool` function, so we should use this kind of notation for all of the type conversions from strings.

Reply:
@ mkcor that is exactly the spirit here :) :+1: